packages/sx05re/emuelec/bin/setres.sh

```diff
@@ -21,6 +21,11 @@
 # 1080i50hz
 # 576cvbs
 
+show_blank()
+{
+  mpv -fs "/storage/.config/splash/blank.png" > /dev/null 2>&1
```

Comment:
please put this in show_splash.sh as not all devices use mpv.

Reply:
Rather than modding show_splash it has simple changes to work on diff devices.

Reply:
Hey man, I decided not to be lazy and fix it so it uses show_splash like you originally said.